pkg/gds/store/trtl/iterator.go

```diff
@@ -0,0 +1,281 @@
+package trtl
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	"github.com/rs/zerolog/log"
+	trtlpb "github.com/trisacrypto/directory/pkg/trtl/pb/v1"
+	"github.com/trisacrypto/directory/pkg/utils/wire"
+	pb "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
+	"google.golang.org/protobuf/proto"
+)
+
+type vaspIterator struct {
+	trtlIterator
```

Comment:
I wasn't aware that you could embed an interface into a struct - that's very handy!